calcom/cal.com
# fix: default organizer bug in managed event type

## What does this PR do?


Fixes  https://github.com/calcom/cal.com/issues/11920

TODO

- [x] Add E2E Test


BEFORE:- 
<img width="1512" alt="Screenshot 2023-10-16 at 8 03 48 PM" src="https://github.com/calcom/cal.com/assets/53316345/ac595737-09bd-4866-a792-7cd7487eff6e">


AFTER:-

<img width="1512" alt="Screenshot 2023-10-16 at 8 04 04 PM" src="https://github.com/calcom/cal.com/assets/53316345/f045d2fa-bebb-4c3e-95a0-f7d590e2d6d5">


How to reproduce?

1) Create a team and add a member
2) Create a managed event type for that team and assign it to other member.
3) Change the location to organizer default App
4) Book a meeting and check the location



## Type of change
